projects/plugins/boost/composer.json

```
@@ -30,12 +30,15 @@
 		"yoast/phpunit-polyfills": "0.2.0"
 	},
 	"scripts": {
-		"fix": [
+		"phpcs:fix": [
 			"composer --working-dir=../../../ phpcs:fix ./projects/plugins/boost"
 		],
-		"lint": [
+		"phpcs:lint": [
 			"composer --working-dir=../../../ phpcs:lint ./projects/plugins/boost"
 		],
+	  	"phpcs:compatibility": [
```

You have to have some spaces left over here:

```suggestion
		"phpcs:compatibility": [
```

---

Fix it in https://github.com/Automattic/jetpack/pull/20669/commits/cb2d57952fafae511916f77eb97e63b6de7a7652